src/components/types/Todo.ts

```diff
@@ -0,0 +1,9 @@
+import { User } from './User';
+
+export type Todo = {
+  id: number;
+  title: string;
+  completed?: boolean;
+  userId: number;
+  user?: User;
```

Comment:
user can't be optional. It's a required value.

Reply:
its not required as we are checking in TodoInfo if user exists

<img width="404" alt="image" src="https://github.com/mate-academy/react_add-todo-form/assets/113982603/5a5720a7-29d6-47d4-9570-6ba8a6a29a61">